canonical-websites/www.ubuntu.com
# Remove 1710 page

## Done

Remove 1710 page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/1710](http://0.0.0.0:8001/1710)
- Make sure page doesn't exist


## Issue / Card

Fixes #3000
